examples/src/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+# Copyright 2020 Andy Maloney <asmaloney@gmail.com>
+
+project(Examples)
+
+set( APP_EXE ReadImages )
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
These should be set on the target, not globally. See `set_target_properties` in the main cmake file.